tools/install-golangci.sh

```diff
@@ -0,0 +1,388 @@
+#!/bin/sh
+set -e
+# Code generated by godownloader on 2018-06-05T12:04:55Z. DO NOT EDIT.
```

Comment:
Where does this file come from? Looks like a copy from https://github.com/golangci/golangci-lint/blob/master/install.sh

Please note: https://github.com/golangci/golangci-lint/blob/master/LICENSE

Reply:
True, though the installer itself seems to be released under public domain.
Having said that, I couldn't be bothered so I've fixed it differently https://github.com/cosmos/cosmos-sdk/pull/3933/commits/367c5a306f733bdaabe770f2bbc47f60f344b9ab